subprojects/base-services/src/main/java/org/gradle/internal/progress/BuildOperationListener.java

```diff
@@ -15,11 +15,14 @@
  */
 package org.gradle.internal.progress;
 
+import org.gradle.internal.scan.UsedByScanPlugin;
+
 /**
  * A listener that is notified as build operations are executed via a {@link org.gradle.internal.operations.BuildOperationExecutor}.
  *
  * @since 3.5
  */
+@UsedByScanPlugin
```

Comment:
This isn't the case. We used this in Gradle 3.5, but no longer do. As such, it doesn't need this annotation.